vars/getBuildManifestInfo.groovy

```diff
@@ -0,0 +1,11 @@
+String[] call(Map args = [:]) {
+    def lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+    String buildManifestUrl = args.buildManifestUrl ?: BUILD_MANIFEST_URL
```

Comment:
I would require the manifest URL parameter, otherwise it's hard to tell that the job calling this needs `BUILD_MANIFEST_URL`.